mkFit/buildtestMPlex.cc

```diff
@@ -353,10 +353,19 @@ double runBuildingTestPlexCloneEngine(Event& ev, EventOfHits &eoh, MkBuilder& bu
 double runBtbCe_MultiIter(Event& ev, EventOfHits &eoh, MkBuilder& builder)
 {
   double ttime = 0;
+  TrackVec seeds_used;
+  TrackVec seeds1;
 
-  // ??? MIMI - What validation prep / map tasks need to run before we start
-  // screwing up seeds etc?
-
+  //not smart usage of memory
```

Comment:
We need to figure out a way to say "if validation" to avoid doing remapping and seed duplication when we don't need it.
It seems const bool validation_on = (Config::sim_val || Config::quality_val); would be a good start?

Reply:
adding the flag root_val_dumb_cmssw